dev-python/busfactor/busfactor-0.2.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="A bus factor analyzer for Git repositories"
+HOMEPAGE="https://github.com/SOM-Research/busfactor"
+SRC_URI="https://timeraider4u.github.io/distfiles/files/${P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-lang/python:2.7"
+RDEPEND="${DEPEND}
+	~dev-python/gitana-${PV}[${PYTHON_USEDEP}]"
+
+python_install() {
+	distutils-r1_python_install
+	# install executable script in /usr/bin
+	local MY_SITEDIR=$(python_get_sitedir)
+	local MY_SCRIPT="${MY_SITEDIR}/${PN}/bus_factor_gui.sh"
+	echo "${PYTHON} ${MY_SITEDIR}/${PN}/bus_factor_gui.py" \
+		>> "${ED}${MY_SCRIPT}" \
+		|| die "Could not create '${ED}${MY_SCRIPT}"
+	fperms 0755 "${MY_SCRIPT}"
+	dodir /usr/bin
+	dosym "${MY_SCRIPT}" "/usr/bin/busfactor_gui"
```

Comment:
I think it had to do with the path-handling by busfactor/gitana which is not directly compatible with Gentoo's file layout, but I have to check again since I wrote the ebuild in October and cannot remember every detail nowadays.